apps/resources/mill.json

```diff
@@ -1,14 +1,12 @@
 {
-  "mainClass": "mill.main.client.MillClientMain",
   "repositories": [
-    "central"
+    "central",
+    "typesafe:ivy-releases"
   ],
+  "launcherType": "prebuilt",
   "dependencies": [
     "com.lihaoyi::mill-dev:latest.stable"
   ],
-  "properties": {
-    "jna.nosys": "true",
-    "MILL_CLASSPATH": "${java.class.path}",
-    "MILL_VERSION": "${mill-dev.version}"
-  }
+  "name": "mill",
+  "prebuilt": "https://github.com/com-lihaoyi/mill/releases/download/${version}/${version}"
```

Comment:
I'd rather not use a prebuilt JAR here. If it were a binary, it would make sense, but as it's a JAR, we can just have coursier pull the right class path.

Reply:
This isn't a jar, it is the standard mill startup shell script which only changes very seldom (except for the embedded version number).

This is all that is required to bootstrap mill and removes the need for the mill-interactive config. Maybe there is a better way of doing this in coursier - I've only examined this configuration from a cursory perspective.

I'll like to be able to install / update mill from `cs` as it a great way to get the whole ecosystem up and running, however the current configuration doesn't work correctly as it doesn't use the standard startup script and @lefou et al. [refuse](https://github.com/com-lihaoyi/mill/issues/new) to look at mill bugs installed from coursier as `mill -i` etc doesn't work etc, which is understandable.

I understood that the recent changes meant that the standard scripts for sbt could now be used. Can the same be done for mill?